OndaEDFSchemas.jl/LICENSE.md

```diff
@@ -0,0 +1,22 @@
+The OndaEDFSchemas.jl package is licensed under the MIT "Expat" License:
+
+> Copyright (c) 2020: Beacon Biosignals, Inc.
```

Comment:
is `2020` right? I suspect it should be `2020-2022`.

Reply:
hmmm, good q.  Technically this package was only "released" in 2022 so it should probably just be 2022